lib/middleware/errors.js

```diff
@@ -24,14 +25,18 @@ function _applicationError(err, req, res, next) {
 function _authenticationError(err, req, res, next) {
   logger.log('warn', {'Unauthorized': 'Invalid user credentials'});
   logger.log('warn', {'Error stack': err.stack});
-  res.redirect(uriConstructor.redirectUrl(req.originalUrl));
+
+  return fetch(config.LOGOUT_URL, { headers: req.headers })
```

Comment:
Might be so. I was just commenting on the code. But yes, you might be right. We might not need this.